nordicsemi/version.py

```diff
@@ -37,4 +37,4 @@
 
 """ Version definition for nrfutil. """
 
-NRFUTIL_VERSION = "6.0.2"
+NRFUTIL_VERSION = "6.0.3"
```

Comment:
This variable is not used in setup.py for defining the version. I think we should link it there:
https://github.com/homersoft/pc-nrfutil/blob/d7c6c6e8186c22d69f489d85c4d3842406f7ba6c/setup.py#L139
The next package version should be 6.0.7